server/src/server.js

```diff
@@ -205,6 +209,20 @@ app.get("/content/:id/:domain", function (req, res) {
   });
 });
 
+app.get("/images/:id/:domain/:clipId", function (req, res) {
```

Comment:
I am unsure whether we should use the shot id here or not.  There's a danger that you might want to share the image but keep the shot private (since it can easily have unintended extra information).  If you share the image by URL you've implicitly (but not obviously) exposed the shot it came from.